src/lib/ComponentLib.sol

```diff
@@ -75,16 +76,13 @@ library ComponentLib {
      * @param sponsorDomainSeparator The domain separator for the sponsor's signature, or zero for non-exogenous claims.
      * @param typehash               The EIP-712 typehash used for the claim message.
      * @param domainSeparator        The local domain separator.
-     * @param validation             Function pointer to the _validate function.
      */
     function processClaimWithComponents(
         bytes32 claimHash,
         uint256 calldataPointer,
         bytes32 sponsorDomainSeparator,
         bytes32 typehash,
-        bytes32 domainSeparator,
-        function(bytes32, uint96, uint256, bytes32, bytes32, bytes32, uint256[2][] memory) internal returns (address)
```

Comment:
This is a good change, this was a holdover from when we passed either a transfer or a withdrawal function in depending on the claim type (now that's captured per-claimant)